src/routes/about/+page.svelte

```diff
@@ -1,12 +1,142 @@
 <script lang="ts">
+  import Icon from "@iconify/svelte";
+  import About from "$lib/components/About.svelte";
+  import Marker from "$lib/components/Marker.svelte";
 </script>
 
-<div>🤗</div>
+<h2>
+  <Marker>about</Marker>
+</h2>
+
+<div>
+  <About />
+  <article>
+    <h3>MISC について</h3>
+    <p>
+      MISC
+      は、愛知工業大学名電高校で情報システム部として活動する生徒たちのグループです。<br
+      />
+      PC に関わることなら自由に活動でき、個人個人が興味のある分野で活動しています。
+    </p>
+    <h4>例えば…</h4>
+    <div>
+      <div>
+        <p>AI、データサイエンス</p>
+        <span>
+          <Icon icon="devicon:python" height={40} />
+          <Icon icon="devicon:tensorflow" height={40} />
+          <Icon icon="devicon:pytorch" height={40} />
+          <Icon icon="devicon:r" height={40} />
+        </span>
+      </div>
+      <div>
+        <p>Web 制作</p>
+        <span>
+          <Icon icon="devicon:html5" height={40} />
+          <Icon icon="devicon:nodejs-wordmark" height={40} />
+          <Icon icon="devicon:vuejs" height={40} />
+          <Icon icon="devicon:svelte" height={40} />
+        </span>
+      </div>
+      <div>
+        <p>ゲーム制作</p>
+        <span>
+          <Icon icon="devicon:unity" height={40} />
+          <Icon icon="devicon:unrealengine" height={40} />
+          <Icon icon="devicon:csharp" height={40} />
+        </span>
+      </div>
+    </div>
+  </article>
+  <article>
+    <h3>このサイトについて</h3>
+    <p>
+      このサイトは、MISC のホームページとして作成されました。<br
+      />制作にあたり、以下の技術を使用しています。
+    </p>
+    <ul>
+      <li>SvelteKit</li>
+      <li>TypeScript</li>
+      <li>MicroCMS</li>
```

Comment:
```suggestion
      <li>microCMS</li>
```
microCMS の m は小文字です警察 👮